qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -57,6 +58,9 @@ setup()
             ${DATA} ${IMAGE}
     done
 
+    # create another pool
+    ceph osd pool create ${ANOTHER_POOL} 32
```

Comment:
nit: I would use 8 for pg_num. Sometimes it may be too much for vstart.sh cluster, when you already have several pools set up, and the command may fail.